lib/srv/regular/sshserver.go

```diff
@@ -187,6 +187,8 @@ type Server struct {
 
 	// lockWatcher is the server's lock watcher.
 	lockWatcher *services.LockWatcher
+
+	createHostUser bool
```

Comment:
Nit: I know this is an internal field but I'd document it as well for consistency with other fields here.